### DISCUSSION TITLE
tmux double-click to escape issue

### DISCUSSION THREAD

**Discussion by MGlolenstine** [helix-editor/helix#2723] [Q&A] [answered]
I'm used to using Vim inside of tmux so that I can have a terminal window split on the side.

When trying to switch between certain modes, `insert -> visual` for example, I have to press `<ESC>`  twice in a row for it to switch.
It could be just tmux eating my inputs, but I don't have such issues with Vim.

**Comment by sudormrfbin** [comment, marked as answer]
Does https://github.com/helix-editor/helix/wiki/Troubleshooting#when-using-tmux-or-screen-there-is-a-delay-after-hitting-escape-before-its-registered fix your issue?
